app/index.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import { SafeAreaView } from 'react-native-safe-area-context';
 import { PaperProvider, Text } from 'react-native-paper';
+import {Link} from "expo-router";
 
+//This is the root file
 export default function Index() {
     return (
         <PaperProvider>
             <SafeAreaView style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
                     <Text>Open up App.js to start working on your app!</Text>
+                    <Link href={"/drill/"}>/drill/</Link>
+                    <Link href={"/drill/1234"}>/drill/1234 slug</Link>
```

Comment:
If we're certain about using bottom tabs navigation, we can consider doing that here: https://docs.expo.dev/router/advanced/tabs/#dynamic-routes

Similar to what Andy's navigation PR was doing.

Reply:
Ye I agree

Reply:
Although I also want the navbar to appear on some of the tabs when they're active. Maybe we need to extract some index.js files out of the folder for tabs?